examples/plugin/CMakeLists.txt

```diff
@@ -39,6 +40,7 @@ foreach(pluginname
     com_osvr_example_Configured
     com_osvr_example_EyeTracker
     com_osvr_example_Locomotion
+	com_osvr_example_Tracker
```

Comment:
this doesn't line up on GitHub, suspect tabs/spaces mismatch. Try using Atom or Notepad2-mod with visible whitespace.